src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -1,45 +1,27 @@
 import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components/native"
 
 import { Button } from "../Components/Button"
 import { MaxBidPicker } from "../Components/MaxBidPicker"
 import { Title } from "../Components/Title"
 
-interface SelectMaxBidProps {
+import { SelectMaxBid_saleArtwork } from "__generated__/SelectMaxBid_SaleArtwork.graphql"
+
+interface SelectMaxBidProps /* extends ViewProperties */ {
   saleArtworkID: string
+  sale_artwork: SelectMaxBid_saleArtwork
 }
 
-// we are using hard-coded values for now.
-const Bids = [
-  {
-    label: "$35,000 USD",
-    value: 3500000,
-  },
-  {
-    label: "$40,000 USD",
-    value: 4000000,
-  },
-  {
-    label: "$45,000 USD",
-    value: 4500000,
-  },
-  {
-    label: "$50,000 USD",
-    value: 5000000,
-  },
-  {
-    label: "$55,000 USD",
-    value: 5500000,
-  },
-]
-
 export class SelectMaxBid extends React.Component<SelectMaxBidProps> {
   render() {
+    // TODO metaphysics should return formatted values
```

Comment:
We consume this endpoint from many places so if we make an API change we'd have to be cognizant of backwards-compatibility

Reply:
Yeah agreed. [The code](https://github.com/artsy/metaphysics/blob/fa05b4c5194168d7b8ddfd8f6e1ac6216e621305/src/schema/sale_artwork.js#L42) specifies bid increments as a list of floats, which was [added here, two years ago](https://github.com/artsy/metaphysics/commit/12c378ca5a11394c72334a748f7e59eeae2fdf6a#diff-ce3847826391a6fd4d78314bb78f4dd2R218). It would be better, I think, to use [`money`](https://github.com/artsy/metaphysics/blob/fa05b4c5194168d7b8ddfd8f6e1ac6216e621305/src/schema/fields/money.js) to avoid having Emission format the data in any way. Maybe we could add a new field (`bid_increment_moneys` or similar?) and deprecate `bid_increments`? Any thoughts?

Reply:
Formatted values sound 👌! Naming is hard, I might lean towards something like `bid_increments_formatted`? But we can debate on the corresponding metaphysics PR.

Reply:
Okay!